pkg/daemon/ceph/osd/volume.go

```diff
@@ -79,9 +82,26 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 		strconv.Itoa(a.storeConfig.OSDsPerDevice),
 	}...)
 
-	// ceph-volume is soon implementing a parameter to specify the "fast devices", which correspond to the "metadataDevice" from the
-	// crd spec. After that is implemented, we can implement this. In the meantime, we fall back to use rook's partitioning.
+	if a.storeConfig.StoreType == config.Bluestore && a.storeConfig.DatabaseSizeMB > 0 {
+		if a.storeConfig.DatabaseSizeMB < cephVolumeMinDBSize {
+			// ceph-volume will convert this value to ?G. It needs to be > 1G to invoke lvcreate.
+			logger.Infof("skipping databaseSizeMB setting. For it should be larger than 1024MB.")
+		} else {
+			batchArgs = append(batchArgs, []string{
+				databaseSizeFlag,
+				// ceph-volume takes in this value in bytes
+				strconv.FormatInt(int64(a.storeConfig.DatabaseSizeMB)*sizeMB, 10),
```

Comment:
Can you use `MbTob` instead? See: https://github.com/rook/rook/blob/master/pkg/util/display/bytes.go#L68

Reply:
Done. Thanks for the hint. :)